packages/editor/src/components/inputs/StringInput.tsx

```diff
@@ -33,16 +33,27 @@ const inputStyle = {
   margin: 0
 }
 
-interface StyledNumericInputProps {
+interface StyledStringInputProps {
   className?: string
-  onChange?: any
+  onChange?: (e: any) => void
```

Comment:
Please do not type these as `any`, this leads to bugs.

Reply:
the e value?

Reply:
we need this to be any, atleast for the stringInputProps as it is used as a base for other props